launch/tier4_perception_launch/launch/object_recognition/detection/camera_lidar_fusion_based_detection.launch.xml

```diff
@@ -28,6 +28,9 @@
   <arg name="container_name" default="pointcloud_container"/>
   <arg name="use_validator" default="true" description="use obstacle_pointcloud based validator"/>
   <arg name="score_threshold" default="0.45"/>
+  <arg name="centerpoint_model_name" default="centerpoint_tiny" description="options: `centerpoint` or `centerpoint_tiny`"/>
```

Comment:
[Q] Is it still better to use centerpoint_tiny as default?

Reply:
`centerpoint` is better, but this pr is aim to pass the model parameters from launch files to the lidar_centerpoint package.  So I will make another PR to change the default model at the same time as lidar_centerpoint package.

Reply:
OK. Sounds good.